sc/SCVim.sc

```diff
@@ -41,6 +58,10 @@ SCVim {
 			vimPath = vimPath.replace("\n", "");
 		};
 
+		vimServerEnabled = "% --version".format(vimPath).unixCmdGetStdOut.contains("+clientserver") and: {
+			"% --serverlist".format(vimPath).unixCmdGetStdOut.contains(vimServerName.toUpper)
+		};
+
```

Comment:
@grirgz What about posting a warning here if `vimServerEnabled` is `nil` saying that `Document.currentPath()` won't work unless vim has been compiled with clientserver option?